charts/internal/seed-controlplane/charts/cloud-controller-manager/templates/cloud-controller-manager.yaml

```diff
@@ -37,15 +37,15 @@ spec:
       - name: gcp-cloud-controller-manager
         image: {{ index .Values.images "cloud-controller-manager" }}
         imagePullPolicy: IfNotPresent
-        {{- if semverCompare ">= 1.31" .Values.kubernetesVersion }}
```

Comment:
This line should not be removed. The requirements for the ipv6 cluster should instead be moved to v1.31+